Framework/DataHandling/src/LoadILLDiffraction.cpp

```diff
@@ -93,6 +94,22 @@ void LoadILLDiffraction::init() {
   declareProperty(make_unique<WorkspaceProperty<MatrixWorkspace>>(
                       "OutputWorkspace", "", Direction::Output),
                   "The output workspace.");
+  std::vector<std::string> calibrationOptions{"", "Raw", "Calibrated"};
```

Comment:
So the default is "Raw"? Maybe we should have "Auto" as default, which will do what the empty string does now?

Reply:
The default is the empty string. I agree calling this `Auto` would be a bit nicer - makes it clear you should check what is going to happen!